src/SFA.DAS.EmployerIncentives.Web/Views/Apply/SelectApprenticeships.cshtml

```diff
@@ -10,8 +10,7 @@
 <main class="govuk-main-wrapper " id="main-content" role="main">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <form asp-action="SelectApprenticeships">
-
+            <form asp-action="SelectApprenticeships" method="post">
                 @Html.AntiForgeryToken()
 
```

Comment:
I know this isn't part of this PR but my understanding is the 

@Html.AntiForgeryToken()

is added automatically to all dotnet core 2+ views (but I may be wrong)

Reply:
it was added by @gdbarnes upon review.